using/api/pie-controller.md

```diff
@@ -0,0 +1,65 @@
+# PieController 
+
+## ES5 access 
+When you load the controller.js in a script tag, a var _pie_controller is created, to give you access to the controller.
```

Comment:
We shouldn't create a global instance of a controller, there may be multiple instances on a page. also can we not use `_pie_x`? I'd propose anything `pie` related lives under a global `pie` object aka `var myController = new pie.Controller(model, env);`